.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Issue:
```

Comment:
What about creating an approach that makes issue and test declaration optional or less opinionated? I mean, for non-code PRs like documentation PRs, improvement or integrations suggestions, etc.

Reply:
I agree with you, do you have a suggestion for this?

Reply:
I made an update with option to non-technical things, what do you think about?